### PR TITLE
Feat/new audio input component

### PR DESCRIPTION
**Pull request by gabrielforster**
### Description
Create new AudioInput component

### Screenshots
image
image
image

### CODE REVIEW

**Review by victorstassun — APPROVED** [review]
LGTM

File: src/components/Inputs/AudioInput/AudioInput.vue [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+    },
+
+    setupAudio(src) {
+      const audio = new Audio(src);
+      this.audio.element = audio;
+
+      audio.addEventListener('timeupdate', () => {
+        const porcentDuration = (audio.currentTime * 100) / this.duration;
+
+        this.audio.timeline = (porcentDuration / 100) * this.audio.width;
```
Comment (mickaelnn) [inline src/components/Inputs/AudioInput/AudioInput.vue:164]: percent*

File: src/components/Inputs/AudioInput/AudioInput.vue [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+      if (layerX < -(this.audio.width) || layerX > this.audio.width) return;
+
+      if (layerX < 0)
+        layerX = this.layerX + layerX < 0 ? 0 : this.layerX + layerX;
+
+      if (layerX > this.audio.width)
+        layerX = this.layerX + layerX > this.audio.width ? this.audio.width : this.layerX + layerX;
+
+      const porcentDuration = (layerX * 100) / this.audio.width;
+      const duration = (porcentDuration / 100) * this.audio.duration;
```
Comment (mickaelnn) [inline src/components/Inputs/AudioInput/AudioInput.vue:275]: percent*

**Review by mickaelnn — CHANGES_REQUESTED** [review]

File: src/components/Inputs/AudioInput/AudioInput.vue [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+    layerX() {
+      return this.audio.timeline;
+    },
+  },
+
+  mounted() {
+    if (this.startRecording) this.toggleRecorder();
+  },
+
+  breforeDestroy() {
```
[the remark this answers is not in the document]
Comment (gabrielforster) [reply src/components/Inputs/AudioInput/AudioInput.vue:141]: https://www.figma.com/file/EvuzOS2hbbaU4MdUwzKR1k/Atendimento-S%2B?type=design&node-id=2946-148392&t=izbvMnFmCdGrdUKp-0
As in the flow of record a audio on customer-service, we will click a external button to start recording. So the component needs to start recording when it mounts.
This is why of this prop. The prop name is changed now... for better understanding

**Review by mickaelnn — APPROVED** [review]
LGTM

**Review by victorstassun — APPROVED** [review]
LGTM

**Pull request merged**